src/core/ddsi/include/dds/ddsi/ddsi_cfgelems.h

```diff
@@ -1706,19 +1706,7 @@ static struct cfgelem shmem_cfgelems[] = {
       "<p>If you don't want to see any log from shared memory, use <i>off</i> to disable log message.</p>"),
     VALUES(
       "off","fatal","error","warn","info","debug","verbose"
-    )),
```

Comment:
Rather than deleting them outright, I'd prefer to make them deprecated. This is pretty easy: just do `DEPRECATED("SubQueueCapacity")` (etc.) and replace the `pf_int` by `pf_nop` so it is no longer is printed in the configuration dump at the beginning of a trace file.

It is also possible to use `uf_nop` instead of `uf_natint` to completely ignore the value provided in the configuration (it would then no longer detect garbage where an integer is expected, but that's ok). And hopefully removing the `MEMBER` entry works, but I'm not 100% sure about that ... if not, it requires but a small change so if it is worth trying.

Reply:
Deprecated them, but I am not sure whether this is the proper way. In any case, they are not used internally anymore.